homework.md

```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
-
+A programming language used primarily by Web browsers to create a dynamic and interactive experience for the user
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+When people want to show theirs code to others
+People can see that what is the older and new versions of that programme
+Programmers can store their project confidently.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You can join and prepare to download any programme without any changes in main project
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+You can download any programme to your local computer without any changes in main project
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+After the changes you can request a permission to accept these changes from the owner of the programme(master)
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you see changes the history of that programme
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+You send the commits of your local repository to Github
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A place for us to version control, store and share our code.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save your code to Github
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+communicate context about a change to other developers working on that project
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+https://git-scm.com/book/en/v2/Getting-Started-Getting-Help
```

Comment:
I think we are supposed to include the "Git Cheatsheet" in this answer, but good call on the Git help link.